src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -848,8 +851,13 @@ private void NormalizeAssigned(ref LocalState state)
             state.Assigned.EnsureCapacity(nextVariableSlot);
             for (int i = oldNext; i < nextVariableSlot; i++)
             {
-                var id = variableBySlot[i];
-                state.Assigned[i] = (id.ContainingSlot > 0) && state.Assigned[id.ContainingSlot];
+                var assigned = false;
+                if (!_trackClasses)
```

Comment:
>if (!_trackClasses) [](start = 16, length = 19)

This feels suspicious. Why would this affect slots allocated for locals?

Reply:
Replaced with explicit check for `struct`.